### DISCUSSION TITLE
Aspire Dashboard support for custom metric path and port

### DISCUSSION THREAD

**Discussion by firdausng** [microsoft/aspire#4135] [Q&A] [answered]
Hi,

Thanks for this wonderful project.

I am trying to use standalone docker aspire dashboard for my application. I have setup my application to use a custom path and port for Prometheus Scraping Endpoint
```csharp
app.UseOpenTelemetryPrometheusScrapingEndpoint(
            context => context.Request.Path == prometheusOptions.MetricsPath
                       && context.Connection.LocalPort == prometheusOptions.LocalPort);
```

this seems like causing the dashboard not able to pickup the metric. I suspect this is because it is listening to default endpoint and port which is `/metrics`

Is there a way to override this setting in the dashboard environment variable?

**Comment by davidfowl** [comment, marked as answer]
The dashboard doesn't scrape your prometheus endpoint. See https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/telemetry
